### PR TITLE
Fix double loading encounters from FQS in both ZAP and Encounters & N…

### PR DESCRIPTION
**Pull request by sarahmangels**
…otes component

### CODE REVIEW

File: src/fhir/basic.ts [hunk 1]
```diff
@@ -82,38 +82,48 @@ export function useBasic(fqs: FeatureToggle) {
   );
 }
 
+function mapBasics<R extends fhir4.Resource, T extends FHIRModel<R>>(
+  queryData: T[],
+  basicsData: Basic[]
+) {
+  const resources2 = queryData;
```
Comment (melocule) [inline src/fhir/basic.ts:89]: why the 2 here?

**Review by melocule — APPROVED** [review]
The fix works! 

But, and idk if this is related or even an issue, in ZAP for MedicationStatementConnection and ConditionConnection i get two queries-- one with edges and one without

e.g. 
image

**Comment by sarahmangels** [comment]
@melocule the doubles for meds and conditions are because one is getting regular resources and one is getting lens resources so that is expected!!

File: src/fhir/basic.ts [hunk 2]
```diff
@@ -82,38 +82,48 @@ export function useBasic(fqs: FeatureToggle) {
   );
 }
 
+function mapBasics<R extends fhir4.Resource, T extends FHIRModel<R>>(
+  queryData: T[],
+  basicsData: Basic[]
+) {
+  const resources = queryData;
+  const basics = basicsData;
```
Comment (mrosata) [inline src/fhir/basic.ts:90]: why not rename the params?

**Pull request merged**